src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -3,7 +3,10 @@
 import static org.mockito.Mockito.when;
 
 import bio.terra.common.category.Unit;
+import bio.terra.model.EnumerateSnapshotAccessRequest;
+import bio.terra.model.EnumerateSnapshotAccessRequestItem;
 import bio.terra.model.SnapshotAccessRequestResponse;
+import java.util.List;
 import java.util.UUID;
 import org.junit.jupiter.api.Assertions;
```

Comment:
I know that this class already pulled in `Assertions`, but I'll note that the prevailing convention in `jade-data-repo` is to pull in static Hamcrest matchers, rather than the JUnit Assertions:

```
import static org.hamcrest.MatcherAssert.assertThat;
```